plugin/pkg/admission/certificates/entrusting/admission.go

```diff
@@ -0,0 +1,94 @@
+package entrusting
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	genericadmissioninit "k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/klog/v2"
+	api "k8s.io/kubernetes/pkg/apis/certificates"
+	"k8s.io/kubernetes/plugin/pkg/admission/certificates"
+)
+
+const PluginName = "ClusterTrustBundleEntrusting"
```

Comment:
If we don't use the `entrust` verb in the authz check then we should probably change the name of the admission plugin too.

Reply:
Done.